src/components/Post/PostCommentForm.tsx

```diff
@@ -141,33 +251,87 @@ const PostCommentForm: FC<IPostCommentFormProps> = (props) => {
 				size={'large'}
 				id={id}
 			/>
+			{isPosted ? (
```

Comment:
Check design for multisig

![image](https://github.com/polkassembly/polkassembly/assets/65517966/58380f10-c678-4205-b66e-6a0d7c8a15e0)